lottie-ios/Classes/PublicHeaders/LOTAnimationView.h

```diff
@@ -46,9 +46,12 @@ typedef void (^LOTAnimationCompletionBlock)(BOOL animationFinished);
 
 // TODO
 /// Sets a progress from 0 - 1 of the animation. If the animation is playing it will stop and the compeltion block will be called.
+/// The animation progress is in terms of absolute progress of the defined animation and does not
+/// take into account negative speeds.
```

Comment:
Mind changing the wordage here a bit? Currently it seems as if animationProgress does not work if the speed is negative, thats not true.

Something like 'the current progress of the animation in absolute time. EG a value of 0.75 always represents the same point in animaiton regardless of positive or negative speed.'